sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -296,8 +296,9 @@ const pi_uint32 DynamicBatchStartSize = 4;
 
 struct _pi_queue : _pi_object {
   _pi_queue(ze_command_queue_handle_t Queue, pi_context Context,
-            pi_device Device, pi_uint32 BatchSize)
+            pi_device Device, pi_uint32 BatchSize, bool QueueIsInorder)
```

Comment:
I suggest that we be more generic and keep `pi_queue_properties`

Reply:
did you mean instead of bool?

Reply:
yes, i.e. save all properties not just in-order one